packages/backend/src/generated/swagger.json

```diff
@@ -2590,7 +2590,6 @@
                     },
                     "filters": {
                         "properties": {
-                            "tableCalculations": {},
```

Comment:
🤔 why... it shouldn't remove them, unless there's something weird going on with tsoa?

Reply:
Ah!, because `generate-api` doesn't build the types from common. I'm going to fix the command to always build common before running tsoa.

Reply:
PR to fix that https://github.com/lightdash/lightdash/pull/8089